src/lambdas/user-update.ts

```diff
@@ -0,0 +1,55 @@
+import * as AWS from 'aws-sdk'
+
+const TABLE_NAME = process.env.TABLE_NAME || ''
+const PRIMARY_KEY = process.env.PRIMARY_KEY || ''
+
+const RESERVED_RESPONSE = `Error: You're using AWS reserved keywords as attributes`,
+  DYNAMODB_EXECUTION_ERROR = `Error: Execution update, caused a Dynamodb error, please take a look at your CloudWatch Logs.`
+
+const db = new AWS.DynamoDB.DocumentClient()
+
+export const handler = async (event: any = {}): Promise<any> => {
```

Comment:
I think this lambda is not working when I test the `updateUser` mutation in your deployed API. Do you the reason?
![image](https://user-images.githubusercontent.com/46347630/148131997-1309f963-d0fd-4802-8674-9ef0eddc46c2.png)

Reply:
So the update seemed to be successful but the mutation was erroring out because it was not returning the id. The reason for this was the lambda was just returning the updated properties. Since id is never updated it was not being passed back to AppSync. Fixed lambda to return all user properties.